index.html

```diff
@@ -54,26 +54,24 @@ <h1 class="intro-heading">A federal agency is about to end online free speech.</
             <p class="form-header-message">Send the FCC a message:</p>
             <p class="form-header-message"><strong><span class="bold-text">tell them to keep their hands off the Internet.</span></strong></p>
           </header>
-          <form method="post" action="https://queue.fightforthefuture.org/action" accept-charset="UTF-8" class="form-fields-wrapper">
+          <form method="post" action="https://actionnetwork.org/petitions/save-online-free-speech/signatures" accept-charset="UTF-8" class="form-fields-wrapper new_signature" id="new_signature" novalidate="novalidate" target="_blank">
```

Comment:
We definitely want this to be submitting to https://queue.fightforthefuture.org/action, not directly to Action Network, as this is how we handle sending letters to Congress and FCC comments in addition to just the petition signature - the other fields will need to be reverted too, and we'll just be updating the `an_petition_id` field. Sorry for the confusion!